pkg/dependencymagnet/dependencymagnet.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
```

Comment:
Is this where we want to keep this? I've seen in a lot of places to just have a `tools.go` in the root of the project, WDYT?